level_5/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+set(
+	SRC_FILES
+	./main.cpp
+	./lexer.hpp
+	./interpreter.hpp
+)
+set(
```

Comment:
I think there's no need to be so super generic here.

Reply:
Done